doc/ref_arch/openstack/chapters/chapter06.md

```diff
@@ -104,10 +104,23 @@ The following rules govern create, read, update, and delete (CRUD) level access.
 ### 6.3.2 System Hardening
 All infrastructure should undergo system hardening, establishes processes to govern the hardening, and documents to cover at a minimal for the following areas:
 
-#### 6.3.2.1 Function
+#### 6.3.2.1 Password policy
```

Comment:
Should we have a policy on 
- frequency of password changes?
- restriction on the reuse of old passwords?
- restriction on use of common words, common names of places, things, name of the component, user, etc.?

Reply:
@pgoyal01 As it is not testable within CNTT conformance context, we can add into best pratices recommendations. 
I am looking for a common reference document we could add here

Reply:
@goyal The CIS password policy guide is a good document for these recommendations